runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -974,7 +976,7 @@ NNFW_STATUS nnfw_session::set_available_backends(const char *backends)
     if (null_terminating(backends, MAX_BACKEND_NAME_LENGTH) == false)
       return NNFW_STATUS_ERROR;
 
-    auto &options = *_coptions;
+    auto &options = *_coptions[0];
```

Comment:
Currently `set_available_backends` assumes there is only one model. We need to extend this API.

Reply:
Or we can use `set_available_backends` to set same backend list for all models, and introduce API for each model (ex. `set_available_backends_for_model(backends, model_idx)`

Reply:
Also note that set_available_backends are used in nnstreamer : https://github.com/nnstreamer/nnstreamer/blob/173e999ebb6866b08c71913e1402b3335b7d0e84/ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c#L243

Reply:
As I wrote in https://github.com/Samsung/ONE/issues/9328#issuecomment-1165166436, I was thinking of keeping the API, but have 1 additional parameter with default value = 0. Anyway, let's discuss in another issue or PR.

Reply:
My comment is also thinking of keeping current API. I thought current `set_available_backend` is used when user want to use specific backend only for package. (and `set_op_backend` same). And I think this PR is proper place to discuss about this.

Reply:
@hseok-oh We may continue to discuss more in #9328 where @chunseoklee and I discussed 1 week ago. Please leave your opinion or idea there. This is PR about supporting multiple model w/o breaking existing API and behavior. The API design for getting/setting config for multiple model is another beast in my mind. Currently we don't provide such API at all.

Reply:
>The API design for getting/setting config for multiple model is another beast in my mind.

Agree. My suggestion about introducing new API can discuss there.

My comment is we need to decide in this PR about `set_available_backend` and `set_op_backend`'s result if user keep current implementation.

Reply:
By offline discussion, I agree to set model index 0 as default for consistency with other APIs. If we need to support package-level backend setting and op-per-backend setting, we can introduce new API later.